rbac/api/common/problem_json_response.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright 2025 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Common exception handler class."""
+from rest_framework.views import Response
+from typing import Any, Dict
+
+
+class ProblemJsonResponse(Response):
+    """Custom problem+json response wrapper."""
+
+    CONTENT_TYPE = "application/problem+json"
+    STATUS_MAP = {
+        400: {
+            "type": "https://console.redhat.com/api/rbac/v2/exceptions/bad-request",
+            "title": "Bad Request",
+            "detail": "There were validation errors with your request.",
```

Comment:
Again, placeholder, but if we agree on how our problem+json errors should be formatted we could encapsulate that in this service.